Video-block to display videos with titles, rating icons and tags

stankasvecova opened the pull request:
.html file shows how the videos should be displayed, however they are still hard-coded.

In .js file I tried to set it up to be added by javascript, but I struggled to figure out how to do the source of the video.

Shoot what to change, how to go about that - thank you!

stankasvecova committed:
A start

.html file shows how the videos should be displayed, however they are still hard-coded.

In .js file I tried to set it up to be added by javascript, but I struggled to figure out how to do the source of the video.

Shoot what to change, how to go about that - thank you!

stankasvecova committed:
corrected paths to .js and .css

MrTim commented on js/videoBlock.js:
```diff
@@ -0,0 +1,51 @@
+
+let source = ["https://www.youtube.com/embed/Uq-FOOQ1TpE", "https://www.youtube.com/embed/93mctUlDofw", "https://www.youtube.com/embed/1YCGM7FiCGg", "https://www.youtube.com/embed/vPukHebsQeQ", "https://www.youtube.com/embed/h11u3vtcpaY"]
+
+function displayVideos (video) {
```
I can't see where do you call this function. So you need to execute it somewhere. 
And if you want it to have an argument it should be named `videos`, because you want to loop through them.

stankasvecova replied:
True. Writing it at 1am shows :) Thanks

MrTim commented on js/videoBlock.js:
```diff
@@ -0,0 +1,51 @@
+let source = ["https://www.youtube.com/embed/Uq-FOOQ1TpE", "https://www.youtube.com/embed/93mctUlDofw", "https://www.youtube.com/embed/1YCGM7FiCGg", "https://www.youtube.com/embed/vPukHebsQeQ", "https://www.youtube.com/embed/h11u3vtcpaY"]
+
+function displayVideos (video) {
+    for (let i=0; i<video.length; i++ )
```
for-loop should have a body inside curly brackets `{}`

stankasvecova replied:
Of course. Ups :)

MrTim commented on js/videoBlock.js:
```diff
@@ -0,0 +1,51 @@
+function displayVideos (video) {
+    for (let i=0; i<video.length; i++ )
+    let videoEl = document.getElementById("video");
+    video.src = url;
```
where you get this url value from?

stankasvecova replied:
changed it to: `videos.src = source[i].value;`
but not right as not working...

MrTim commented on js/videoBlock.js:
```diff
@@ -0,0 +1,51 @@
+    let videoEl = document.getElementById("video");
+    video.src = url;
+    let videoEl = document.createElement("div");
+    source.setAttribute('src', source[i]);
```
you're looping through `video` variable but working with `source[i]`. You can merge two arrays together and src would be an attribute of video element.

stankasvecova replied:
I tried to reconnect the two arrays together but not sure if correctly (upcoming push). Hm...

MrTim reviewed (changes requested):
Left some pointing comment. First you need to fix those, because at the moment js code is broken.

stankasvecova committed:
a few small bugs fixed...

Not sure if I should have
videos.push(); method there too, or videos.play() would be enough.

And how to have the two arrays combined so they work together.

stankasvecova committed:
progress with videos

stankasvecova committed:
small updates

BTBTravis commented on js/setupFirebase.js:
```diff
@@ -0,0 +1,33 @@
+const firebaseConfig = {
```
this pull request is about video blocks please remove firebase related content

stankasvecova replied:
All removed now.

BTBTravis commented on js/signupsignin.js:
```diff
@@ -0,0 +1,52 @@
+const signUpEl = document.getElementById("signUpBtn");
```
should not be in this PR

stankasvecova replied:
All removed now.

BTBTravis commented on js/videoBlock.js:
```diff
@@ -0,0 +1,58 @@
+const videos = [
+    {
+        title: "Miss Bee, a teen singer with a beautiful voice: High hopes",
```
👍 looking good. I do think we'll be able to get title and video id dynamically eventually so this is great data to start with.

BTBTravis reviewed (changes requested):
Video part is looking good please remove non video stuff and we can merge

stankasvecova committed:
Delete signupsignin.js

stankasvecova committed:
Delete setupFirebase.js

stankasvecova committed:
Delete registration.js

stankasvecova committed:
Delete registration.css

stankasvecova committed:
Delete registration.html

stankasvecova commented:
Removed the files you requested, think this PR is ready to merge. Please approve. Thnx.

BTBTravis commented:
I don't think we are on the same page with the docs folder. 

The docs folder is for project documentation. Basically github stuff, things like markdown files and the images used in these markdown files. Its is not really for project assets like `docs/medal-first-place.gif` those should be in a assets folder or something not docs

BTBTravis reviewed (approved):
Beside for non documentation images in the docs folder I'm cool with it
